tools/run_tests/performance/scenario_config.py

```diff
@@ -456,6 +456,145 @@ def scenarios(self):
   def __str__(self):
     return 'node'
 
+class PythonLanguage:
+
+  def __init__(self):
+    self.safename = 'python'
+
+  def worker_cmdline(self):
+    return ['tools/run_tests/performance/run_worker_python.sh']
+
+  def worker_port_offset(self):
+    return 500
+
+  def scenarios(self):
+    for secure in [True]:
+      if secure:
+        secstr = 'secure'
+        secargs = {'use_test_ca': True,
```

Comment:
nit: now there's  a constant for  secure secargs and we are only running secure configuration for all wrapped languages.